### PR TITLE
Add a prereq section on what to know before doing Scan 2.0

### PR DESCRIPTION
**Pull request by rxl7906**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
1.6

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

**Review by chaomonica — APPROVED** [review]

File: scst-scan/bring-your-own-scanner.hbs.md [hunk 1]
```diff
@@ -16,3 +16,17 @@ You might have an existing investment in a scan solution that VMware does not ha
 1. [Create an ImageVulnerabilityScan](ivs-create-your-own.hbs.md): Create an ImageVulnerabilityScan template that tells the Tanzu Application Platform how to run your scanner.
 2. [Verify your ImageVulnerabilityScan](verify-app-scanning.hbs.md): Verify that your ImageVulnerabilityScan is working correctly so that downstream Tanzu Application Platform services work correctly.
 3. [Wrap your ImageVulnerabilityScan in a ClusterImageTemplate](clusterimagetemplates.hbs.md): The ClusterImageTemplate wraps the ImageVulnerabilityScan and allows the Tanzu Application Platform supply chain to run the scan job.
+
+## <a id="prerequisities"></a>Prerequisites
+
+TAP users will need to:
+
+- Provide their own Vulnerability Scanner Image either by:
+  - Using a publicly available image with the scanner CLI.
```
Comment (ryan-a-baker) [inline scst-scan/bring-your-own-scanner.hbs.md:25]:
```suggestion
  - Using a publicly available image that contains the scanner CLI.
```

File: scst-scan/bring-your-own-scanner.hbs.md [hunk 2]
```diff
@@ -16,3 +16,17 @@ You might have an existing investment in a scan solution that VMware does not ha
...
+    - For example, this image with Trivy scanner CLI from [Dockerhub](https://hub.docker.com/r/aquasec/trivy/tags).
```
Comment (ryan-a-baker) [inline scst-scan/bring-your-own-scanner.hbs.md:26]:
```suggestion
    - For example, the official image from Aqua for Trivy from [Dockerhub](https://hub.docker.com/r/aquasec/trivy/tags).
```

File: scst-scan/bring-your-own-scanner.hbs.md [hunk 3]
```diff
@@ -16,3 +16,17 @@ You might have an existing investment in a scan solution that VMware does not ha
...
+  - Building their own image with the scanner CLI which allows for:
+    - More customizable scanning experience.
+      - For example, creating an image with the scanner CLI along with any required dependencies to run the scanner CLI.
+    - Managing vulnerabilities within the TAP user's custom image.
```
Comment (ryan-a-baker) [inline scst-scan/bring-your-own-scanner.hbs.md:30]: Can you elaborate on this a bit?  I'm not sure what you are referring to.  Does this mean patching vulnerabilities that exist in it?

File: scst-scan/bring-your-own-scanner.hbs.md [hunk 4]
```diff
@@ -16,3 +16,17 @@ You might have an existing investment in a scan solution that VMware does not ha
...
+    - Managing their own image to meet the TAP user's legal compliance standards.
```
Comment (ryan-a-baker) [inline scst-scan/bring-your-own-scanner.hbs.md:31]:
```suggestion
    - Managing their own image to meet the TAP user's compliance standards.
```
I like this, but I'd just say compliance standards.  I don't think we need to bring legal in to it :)

File: scst-scan/bring-your-own-scanner.hbs.md (hunk 3 above)
Comment (rxl7906) [reply scst-scan/bring-your-own-scanner.hbs.md:30]: Yes I meant patching vulnerabilities that existing the image which was an issue for us in Scan 1.0. But if the TAP user's build their own image, they would have more control on addressing the vulnerabilities that exist in it.

Comment (rxl7906) [reply scst-scan/bring-your-own-scanner.hbs.md]: I removed this line as requested.

File: scst-scan/bring-your-own-scanner.hbs.md [hunk 5]
```diff
@@ -16,3 +16,16 @@ You might have an existing investment in a scan solution that VMware does not ha
...
 2. [Verify your ImageVulnerabilityScan](verify-app-scanning.hbs.md): Verify that your ImageVulnerabilityScan is working correctly so that downstream Tanzu Application Platform services work correctly.
 3. [Wrap your ImageVulnerabilityScan in a ClusterImageTemplate](clusterimagetemplates.hbs.md): The ClusterImageTemplate wraps the ImageVulnerabilityScan and allows the Tanzu Application Platform supply chain to run the scan job.
+
+## <a id="prerequisities"></a>Prerequisites
+
+TAP users will need to:
+
+- Provide their own Vulnerability Scanner Image either by:
+  - Using a publicly available image that contains the scanner CLI.
+    - For example, the official image from Aqua for Trivy from [Dockerhub](https://hub.docker.com/r/aquasec/trivy/tags).
```
Comment (ryan-a-baker) [inline scst-scan/bring-your-own-scanner.hbs.md:26]:
```suggestion
    - For example, the official Aqua image for Trivy from [Dockerhub](https://hub.docker.com/r/aquasec/trivy/tags).
```

**Pull request merged**